components/editor/modules/dynamiccomponent/index.js

```diff
@@ -92,6 +93,11 @@ const DynamicComponent = ({ rule, subModules, TYPE }) => {
               showException
               key={JSON.stringify(data)}
               require={dynamicComponentRequire}
+              identifiers={
+                data.identifier && {
+                  [data.identifier]: Placeholder(data.identifier)
+                }
+              }
```

Comment:
When Placeholder is a regular component do the following:
```suggestion
              identifiers={
                data.identifier && {
                  [data.identifier]: Placeholder
                }
              }
```
and see comment below to map in identifier